azure-functions-powershell-library/BundledBindings/ActivityTriggerBinding.cs

```diff
@@ -0,0 +1,26 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.Azure.Functions.PowerShellWorker;
+using System.Management.Automation.Language;
+
+namespace AzureFunctions.PowerShell.SDK.BundledBindings
+{
+    public class ActivityTriggerBinding : IInputBinding
+    {
+        public override string BindingAttributeName => "ActivityTrigger";
+
+        public override string BindingType => "activityTrigger";
+
+        public override BindingInformation ExtractBinding(AttributeAst attribute, ParameterAst parameter)
+        {
+            BindingInformation bindingInformation = new BindingInformation();
+            bindingInformation.Name = parameter.Name.VariablePath.UserPath;
```

Comment:
Yeah - sorry this took me so long to get to, I didn't realize that GitHub only shows you a limited number of the conversations each user starts. 

So, for example, this binding: 
    [Function()]
    param(
        [HttpTrigger('anonymous', ('get', 'post'), 'DurableStart')]
        $Request, 
        $TriggerMetadata
    )
The "name" of this binding is Request, because it is tied to the $Request parameter of the param block. If we change the parameter name, we change the name of the binding so that the worker can pass the request info to the correct parameter.